### ISSUE TITLE
Error: Could not find video device with name [screen-capture-recorder] among source devices of type video

### ISSUE THREAD

**Issue by mih-kopylov** [ShareX/ShareX#4609]
Version: 13.0.1
Windows 10 1903 build 18362.592

Steps To Reproduce
* click menu item Capture - Record video
* click to select region 

Actual
* console appears with error
```
ffmpeg version 4.2 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 9.1.1 (GCC) 20190807
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-libmfx --enable-amf --enable-ffnvcodec --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth --enable-libopenmpt
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
[dshow @ 047099c0] Could not find video device with name [screen-capture-recorder] among source devices of type video.
video=screen-capture-recorder:audio=virtual-audio-capturer: I/O error
```

Expected
* video recording stars

**Comment by Fuzzqt** [comment]
I've had this issue with using Install Recorder Device. Make sure your hotkey is set to Screen Record Start/Stop screen record using custom region.

Also another big thing I missed was one of my audio programs that handles my sound from Asus was conflicting with the software. So check your video/audio programs you use and try closing the ones you don't need to see if this fixes it. You can simply do this by Screen Recording Options > Test with CMD to reproduce after every closed application. Also make sure your video drivers are up to date and then re-install and download the ffmpeg and recorder device if needed.

If these don't work try messing with the codecs (shouldn't be required but something worth trying) for me it was applications that conflicted with my video/sound so first step is definitely check those and maybe close the running app or try disabling them in device manager. 

If none of these work please post back with further results! I'm sure we'll figure something out.

**Comment by mih-kopylov** [comment]
I've cheked hotkeys: shift + printscreen is set there. I've checked with Ctrl + Alt + Printscreen, but it didn't help. Then I changed the hotkey back.

I have closed all my applications, it also didn't help. 
I have found installed, but not running screencast-o-matic v2, uninstalled it, but it didn't help.

Then I have opened "Screen Recording Options" and realized that in Sources block there was nothing selected in Video dropdown, so once I have selectected "GDI Grab" value, it started working.
So that helped me.

Thank you very much!
